climada/engine/impact.py

```diff
@@ -1372,7 +1372,147 @@ def _selected_exposures_idx(self, coord_exp):
             LOGGER.warning("No exposure coordinates match the selection.")
         return sel_exp
 
+    @classmethod
+    def concat(cls, imp_list,concat_type='time_window'):
+        """Concatenate impact objects of the same hazard type, frequency, 
+        exposure, and coordinate system.
 
+        Parameters
+        ----------
+        imp_list : list of climada.engine.Impact objects
+                list of Impact object to concatenate
+        concat_type: string
+                string specifying the type of concatenation to be done.
+                    'time_window': concatenate impacts for different time windows / time slices within a year (e.g. May to June, July to August,...)
```

Comment:
I think this is a bit of a confusing naming. I would expect that with the value `time_window`, I would have to actually define a time window over which the algorithm will aggregate in some way.

How about `event` ?